TTHAnalysis/python/plotter/ttH-multilepton/mca-includes/mca-2lss-sigprompt.txt

```diff
@@ -20,10 +28,24 @@ EWK 	: WJetsToLNu_LO : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRig
 #EWK	: W3JetsToLNu_LO_skimSameSign : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ;  FillColor=ROOT.kViolet-4
 #EWK	: W4JetsToLNu_LO_skimSameSign : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ;  FillColor=ROOT.kViolet-4
 EWK     : ZZTo4L : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ;  FillColor=ROOT.kViolet-4
-EWK     : DYJetsToLL_M10to50_LO : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kViolet-4
-EWK     : DYJetsToLL_M50 : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kViolet-4
-
+EWK     : DYJetsToLL_M10to50_LO : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kViolet-4 
+EWK     : DYJetsToLL_M50 : xsec : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge ; FillColor=ROOT.kViolet-4 
 
+<<<<<<< HEAD
```

Comment:
Conflicts must be solved!!!!!!!!!!!!!! Here is a link to the specific part of the git book: https://git-scm.com/docs/git-merge

Reply:
resolved